phpunit.xml.dist

```diff
@@ -76,7 +77,7 @@
             <file>tests/StubsForbiddenTypeHintsTest.php</file>
             <file>tests/StubsPhp81Tests.php</file>
         </testsuite>
-        <testsuite name="PHP_8.2alpha">
+        <testsuite name="PHP_8.2">
```

Comment:
I think that it also makes sense to remove StubsConstantsAndParametersValuesTest from 8.1 suite and leave it only on 8.2 suite since constant values change on every release but we support values check only against the latest release